docs/wiki/basic-tutorials/interactions/responding.md

```diff
@@ -23,6 +23,11 @@ event.getInteraction()
 ::: tip INFO
 Note that you have to respond withing 3 seconds, or the command will fail. If you need longer than 3 seconds you have to
 respond with `respondLater()` which allows you to respond within 15 minutes.
+
+Because of this time limitation, sending any files when using `createImmediateResponder` is not possible.
+It's best to use the `respondLater` functionality of interactions.
+In case you really don't want to use `respondLater`. You can send a web link of the image.
+You can use the methodes that use online hosting. Like discords own cdn an example link is this:`https://cdn.discordapp.com/embed/avatars/0.png`.
```

Comment:
```suggestion
Because of this time limitation, sending any files when creating an immediate response is not possible.
If you want a file to be embedded either use `respondLater` or include a web link in the message content.
Depending on the media type of the link and the server configuration, Discord will then display an appropriate embed for the file.
```
It still was a bit misleading and had mostly chopped of half sentences. Do you feel this suggestion still transports what you were trying to express?

Reply:
jep, alright seems good :)